instance/tests/models/test_instance.py

```diff
@@ -22,13 +22,13 @@
 
 # Imports #####################################################################
 
+from unittest.mock import call, patch, Mock
 from urllib.parse import urlparse
 
 import re
 import yaml
```

Comment:
@antoviaque Seems like these imports aren't in the right place - `re` should go above `from unittest.mock ...`, and `yaml` should probably go below the `django` imports.